test/wallet-auction-test.js

```diff
@@ -123,6 +124,17 @@ describe('Wallet Auction', function() {
     }
   });
 
+  it('should fail to send bid to null address', async () => {
+    const mtx = await winner.makeBid(NAME1, 1000, 2000, 0);
+    mtx.outputs[0].address = new Address();
+    await winner.fill(mtx);
+    await winner.finalize(mtx);
+
+    const fn = async () => await winner.sendMTX(mtx);
+
+    assert.rejects(fn, {message: 'Cannot send to null address.'});
```

Comment:
I changed the error message to verify that it does indeed fail when the message is different.

Reply:
You should add an `await` here, it still works without it but we have seen race conditions in the past without it. It fails even without an await when the error message doesn't match.

Reply:
thanks will fix